search/src/Components/upload/upload_stepper.js

```diff
@@ -197,6 +220,26 @@ class UploadStepper extends React.Component {
         })
     }
 
+    handleUploadMetadata(){
+        axios({
+            method: 'post',
+            url: baseURL + "api/upload_metadata/",
+            data: {
+                "upload_id": this.state.upload_id,
+                "metadata": this.state.metadata
+            },
+            headers: {'X-CSRFToken': csrftoken }
+        }).then(res => {
+            console.log(res)
+            this.handleErrorMessage("")
+            this.handleNext()
+        })
+        .catch(err => {
+            console.log(err)
+            this.handleErrorMessage("Invalid input for Metadata")
```

Comment:
I don't think there's any reason to believe the failure was due to the input

```suggestion
            this.handleErrorMessage("Failed to submit metadata")
```